.github/workflows/humble-source-build.yml

```
@@ -0,0 +1,41 @@
+name: Humble Source Build
+on:
+  schedule:
+    # Run every morning to detect flakiness and broken dependencies
+    - cron: '43 1 * * *'
+
+jobs:
+  humble_source:
+    runs-on: ubuntu-20.04
```

Could that also be

```suggestion
    runs-on: ubuntu-22.04
```
or is that somehow less stable as host OS?

---

It does not really matter as everything relevant is run inside of a container anyway. We could change it, but i would only do that if we change the corresponding workflow(s) on the humble branch at the same time to avoid inconsistencies and i don't see any reason to do so now.